day-four-challenge/secure-container/getPasswords.js

```diff
@@ -1,34 +1,69 @@
-const { map, sort, not, uniq, equals, unfold, allPass, pipe } = require('ramda');
+const { map, sort, not, uniq, allPass, join, replace, range, subtract } = require('ramda');
 
 const MIN = 359282;
 const MAX = 820401;
-const PASSWORD_MIN_LENGTH = 6;
-
-const diff = (a, b) => { return a - b };
+const PASSWORD_LENGTH = 6;
 
+/** Determines if an array is sorted
+ * @param {Array.Number} array 
+ * @returns {Boolean}
+ */
 const digitsIncreaseOrStayTheSameFromLeftToRight = array => (
-  equals(sort(diff, array), array)
-);
+  sort((a, b) => subtract(a, b), array).toString() === array.toString());
+
+/** Determines if an array has at least two repeating digits by teesting that it is not unique
```

Comment:
Just a typo ('teesting')
```suggestion
/** 
 * Determines if an array has at least two repeating digits by testing that it is not unique
```